File: app/src/main/java/com/worldventures/dreamtrips/social/ui/profile/service/analytics/ViewMyProfileAdobeAnalyticAction.java
package com.worldventures.dreamtrips.social.ui.profile.service.analytics;

import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.core.service.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.Attribute;
import com.worldventures.core.service.analytics.BaseAnalyticsAction;

@AnalyticsEvent(action = "profile",
                trackers = AdobeTracker.TRACKER_KEY)
public class ViewMyProfileAdobeAnalyticAction extends BaseAnalyticsAction {

   @Attribute("view") final String view = "1";

}

File: app/src/main/java/com/worldventures/dreamtrips/social/service/profile/analytics/ViewMyProfileAdobeAnalyticAction.kt
package com.worldventures.dreamtrips.social.service.profile.analytics

import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.core.service.analytics.Attribute
import com.worldventures.core.service.analytics.BaseAnalyticsAction
import com.worldventures.janet.analytics.AnalyticsEvent

@AnalyticsEvent(action = "profile", trackers = [(AdobeTracker.TRACKER_KEY)])
class ViewMyProfileAdobeAnalyticAction(@Attribute("view") val view: String = "1") : BaseAnalyticsAction()
